common/flatpak-dir.c

```diff
@@ -3209,18 +3209,26 @@ flatpak_dir_do_resolve_p2p_refs (FlatpakDir             *self,
   g_auto(GLnxLockFile) child_repo_lock = { 0, };
   g_autoptr(GFile) user_cache_dir = NULL;
   g_autoptr(FlatpakTempDir) child_repo_tmp_dir = NULL;
+  g_autoptr(GString) refs_str = NULL;
```

Comment:
Nitpick: Typo “it's not an erorr” in the commit description for “common: Properly handle empty arrays of P2P results”.